config/locales/en.yml

```diff
@@ -1792,7 +1792,7 @@ en:
         capital: disposable capital
     legislation_table:
       paragraph_1_html: This calculation was made using the rules for controlled work in <a href="https://www.legislation.gov.uk/uksi/2013/480/contents" target='_blank' rel='noreferrer noopener'>The Civil Legal Aid (Financial Resources and Payment for Services) Regulations 2013 (opens in new tab)</a> and the <a href="https://www.legislation.gov.uk/all?title=Civil%20Legal%20Aid%20%28Financial%20Resources%20" target='_blank' rel='noreferrer noopener' aria-label="Link opens a search page which includes a list of amendments to The Civil Legal Aid (Financial Resources and Payment for Services) Regulations 2013 (opens in new tab)">amendments to these Regulations (opens in new tab)</a>.
-      paragraph_2_html: Guidance on determining financial eligibility for controlled work and family mediation can be found in the <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/983065/Lord_Chancellor_s_guidance_on_determining_controlled_work_and_family_mediation.pdf" target='_blank' rel='noreferrer noopener'>guide to determining controlled work (opens in new tab)</a>.
+      paragraph_2_html: Guidance on determining financial eligibility for controlled work and family mediation can be found in the <a href= %{page_url} target='_blank' rel='noreferrer noopener'>guide to determining controlled work (opens in new tab)</a>.
```

Comment:
Here you've got `<a href= %{page_url} target...`, which is going to result in:

```html
<a href= https://assets.blah...
```

when you want 

```html
<a href="https://assets.blah...
```

So I think you want to remove the space after `href` and put quotation marks around `%{page_url}`. The browser might be able to cope with the space and the lack of quotation marks and still interpret the HTML correctly, but it's a bit iffy.